analyze.py

```diff
@@ -143,14 +191,54 @@ def _serialize_json(self, data):
        return json.dumps(data, indent=4)
    
     def _serialize_csv(self, data):
-       return json.dumps(data, indent=3) #fixme - make this export csv instead of json
-
+        with open('output.csv', 'w', newline='') as csvfile:
+            csvwriter = csv.writer(csvfile, delimiter=',',quotechar=',', quoting=csv.QUOTE_NONE,escapechar="\\")	
+            csvwriter.writerow(['']+['students_per_category'])
+            csvwriter.writerow(['']+['round_1']+['round_2']+['round_3'])
+            SpC = analayzer.worksheet["students_per_category"] #SpC = Students per category
+            categories = analayzer.category_amount
+            for i in range(categories):
```

Comment:
single letter lower case i is as bad as a lower case l , as you can see its hard to see the difference when reading the code, thats why